lib/rspec-gc-control/base_text_formatter.rb

```diff
@@ -1,45 +1,50 @@
+require 'rspec/core/version'
 require 'rspec/core/formatters/base_formatter'
 
-module RSpec
-  module Core
-    module Formatters
+if defined?(::RSpec::Core) && ::RSpec::Core::Version::STRING < '3.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.